charts/zabbix/docs/example/kind/values.yaml

```diff
@@ -89,7 +89,7 @@ zabbixweb:
   ZBX_SERVER_HOST: zabbix-zabbix-server
   ZBX_SERVER_PORT: 10051
   service:
-    type: ClusterIP
+    type: NodePort
```

Comment:
Same as above regarding NodePort service

Reply:
Hi @sa-ChristianAnton!

To use ingress, the service must be NodePort. I switched to make it easier. But I understand your concerns and have rolled back.